tests/test_preprints.py

```diff
@@ -140,6 +140,29 @@ def test_non_admin_update_file(self):
             self.preprint.set_primary_file(self.file_two, auth=self.read_write_user_auth, save=True)
         assert_equal(self.project.preprint_file._id, self.file._id)
 
+    def test_public_private_preprint_banner_on_file_exist(self):
```

Comment:
@brianjgeiger IIRC, I put it here because the webtests didn't have any of the preprint relevant boilerplate needed. I can transfer move it if it belongs there.